MonoGame.Framework/BoundingBox.cs

```diff
@@ -352,38 +448,60 @@ public override int GetHashCode()
             return this.Min.GetHashCode() + this.Max.GetHashCode();
         }
 
+        /// <summary>
+        /// Determines if this <see cref="BoundingBox"/> intersects with another.
+        /// </summary>
+        /// <param name="box">The box to test against</param>
+        /// <returns>True if the boxes intersect, false otherwise</returns>
         public bool Intersects(BoundingBox box)
         {
             bool result;
             Intersects(ref box, out result);
             return result;
         }
 
+        /// <summary>
+        /// Determines if this <see cref="BoundingBox"/> intersects with another, placing the result in the "result" parameter
+        /// </summary>
+        /// <param name="box">The box to test against</param>
+        /// <param name="result">The boolean in which to place the result</param>
         public void Intersects(ref BoundingBox box, out bool result)
         {
             if ((this.Max.X >= box.Min.X) && (this.Min.X <= box.Max.X))
             {
-                if ((this.Max.Y < box.Min.Y) || (this.Min.Y > box.Max.Y))
+                if ((this.Max.Y >= box.Min.Y) && (this.Min.Y <= box.Max.Y))

```

Comment:
First it is preferable not mix documentation work with changes to the code.  It makes it harder for us to check and approve pull requests.

Second we really don't want to see "code cleanup" like changes.  These are changes where you just shuffle code around but have no actual change to functionality.  This just makes maintenance of the project harder.

You can see more about what we expect in pull requests here:

https://github.com/MonoGame/MonoGame/blob/develop/CONTRIBUTING.md#quick-guidelines

So unless the changes in this function are an actual bug fix or performance improvement...  I suggest reverting these changes.